test/abort/test-signal-handler.js

```diff
@@ -17,8 +17,11 @@ if (process.argv[2] === 'child') {
   const child = spawnSync(process.execPath,
                           ['--expose-internals', __filename, 'child'],
                           { stdio: 'inherit' });
-  // FreeBSD uses SIGILL for this kind of crash.
+  // FreeBSD uses SIGILL (v12.2) or SIGBUS (v12.4) for this kind of crash.
```

Comment:
Seeing this on `v13.1` as well.
```suggestion
  // FreeBSD uses SIGILL (v12.2) or SIGBUS (v12.4 and greater) for this kind of crash.
```